lib/omnibus/logger.rb

```diff
@@ -135,7 +135,7 @@ def format_message(severity, progname, message)
       else
         left = "#{format_severity(severity)} | "
       end
-      "#{left.rjust(LEFT)}#{message}\n"
+      "#{left.rjust(LEFT)}#{Time.now} | #{message}\n"
```

Comment:
Looks like changing the output broke a few of our assertions so the tests are failing.  But I'm really glad to have the time in the logs now, it will really help debugging.